examples/zcentral_remote_boost_linux/config.json

```diff
@@ -0,0 +1,102 @@
+{
+    "location": "variables.location",
+    "resource_group": "variables.resource_group",
+    "install_from": "headnode",
+    "admin_user": "hpcadmin",
+    "variables": {
+        "hpc_image": "OpenLogic:CentOS:7_8:latest",
+        "location": "<NOT-SET>",
+        "resource_group": "<NOT-SET>",
+        "vm_type": "Standard_NV12s_v3",
+        "low_priority": false,
+        "vnet_resource_group": "variables.resource_group",
+        "zcentral_remote_boost_sender_sas_url": "<NOT-SET>",
```

Comment:
you can probably simplify this and use the sasurl.<STORAGE-ACCOUNT>.<STORAGE-PATH> syntax to let azhpc generate the sas key for you and have variables for storage-account and path. The syntax would be then :
sasurl.{{variables.storage_account}}.{{variables.storage_path}}

Reply:
I already assume that the sasurl macro will be used for "zcentral_remote_boost_sender_sas_url" and "zcentral_remote_boost_sender_license_sas_url"

Reply:
Great, then having the pattern to use in the value instead of NOT-SET will guide the end user on how to use it

Reply:
Done.